app/com/linkedin/drelephant/util/InfoExtractor.java

```diff
@@ -116,6 +116,13 @@ public static void loadInfo(AppResult result, HadoopApplicationData data) {
     } else if ( data instanceof SparkApplicationData) {
       properties = retrieveSparkProperties((SparkApplicationData) data);
     }
+
+    // TODO temporary fix, it should be automatically set by langoustine in the /conf properties
```

Comment:
This has been removed, as I am now retrieving the project name in a non hacky way, from the /conf properties.This should currently works for all Yarn/Hive job scheduled by Langoustine, as long as the project name is properly defined in the configuration of the project (reference.conf).